reflex/components/datadisplay/logo.py

```diff
@@ -0,0 +1,21 @@
+from reflex.components.component import Component
+import reflex as rx
+
+class Logo(Component):
+
+    @classmethod
+    def create(cls, **props,):
+        return rx.center(
+        rx.link(
+            rx.hstack(
+                "Built with ",
+                rx.image(src="https://raw.githubusercontent.com/reflex-dev/reflex-web/main/assets/Reflex.svg"),
```

Comment:
I'd rather we add the logo in the source than using a link.

Reply:
We currently don't have a way to bundle up assets with a component (unless we include the logos in every app by default, which we probably don't want)

Reply:
Yeah but using a link means we'll break some version of Reflex in the future if we ever move the image somewhere.
Could we make a NPM package that is a dependency of Logo ? 
This way the image would be only installed when the component is used?

Reply:
Put it into html and then inside a color mode cond

Reply:
done